backend/src/lastwill/utilities/grammar.utils.ts

```diff
@@ -0,0 +1,28 @@
+import { Gender, PersonType } from '../../db/entities/lastwill.entity'
+
+export function getPossessivePronouns(
```

Comment:
I mean yes that works but coulnt we replace that with a simple map that has as type
[key as String]: TheEnum
=> Map
Persons
{PersonType.CHILD: "Mein Kind", ...}
Childs
{Gender.FEMALE: 'Meine Schwester',...}

Then u could also ignore the tests in the util-spec

=> Also just as a thought, just replacing all genders automatically with neutrals
Father/Mother to "Elternteil" since I think it is weird to say Father 1 and Father 2 instead of Parent 1 and 2
Brother/Sister to "Geschwisterkind"
other to "Bekannter"
other can be Related or not related, so "Verwandter" is kinda not accurate, but "Bekannter" could also be a related person and thus works

Reply:
"Bekannter" is less accurate than verwandter...and all other types in the array are family, so the default is also family.

And no switch > enum in this case